.github/workflows/test.yml

```diff
@@ -23,7 +23,8 @@ jobs:
     - name: Install dependencies
       run: |
         python -m pip install --upgrade pip
-        python -m pip install tox-gh-actions poetry
+        python -m pip install poetry
         poetry install
+        poetry run python -m pip install tox-gh-actions
```

Comment:
Another fixing:

previously the `tox-gh-actions` plugin is not recognized and all tox environments are running

ex. last #11 PR's checks: https://github.com/m3dev/kannon/actions/runs/4649331048/jobs/8227600026?pr=11

```
Run poetry run tox

py38: SKIP ⚠ in 0.88 seconds
py39: commands[0]> coverage run -m unittest discover -s test
py39: OK ✔ in 6.63 seconds
py310: commands[0]> coverage run -m unittest discover -s test
py310: OK ✔ in 2.82 seconds
py311: SKIP ⚠ in 0.01 seconds
pyyapf: commands[0]> coverage run -m unittest discover -s test
pyyapf: OK ✔ in 2.68 seconds
pyisort: commands[0]> coverage run -m unittest discover -s test
pyisort: OK ✔ in 2.68 seconds
```

now: https://github.com/m3dev/kannon/actions/runs/4668051726/jobs/8264603602?pr=13

```
Run poetry run tox
py39: freeze> python -m pip freeze --all
py39: pip==23.0.1,setuptools==67.4.0,wheel==0.38.4
tox: py39
  py39: OK (8.03=setup[1.21]+cmd[6.82] seconds)
  congratulations :) (8.46 seconds)
```